src/app/form-builder/components/header/header-layout/header-layout.component.html

```diff
@@ -0,0 +1,28 @@
+<mat-toolbar color="primary">
+  <nav class="nav">
+    <div *ngIf="!isAuthenticated" style="padding: .5rem">
```

Comment:
replace inline styles with css classes
```suggestion
    <div *ngIf="!isAuthenticated" style="padding: .5rem">
```